pkgs/top-level/all-packages.nix

```diff
@@ -37917,7 +37917,11 @@ with pkgs;
     branch = "canary";
   };
 
-  golden-cheetah = libsForQt5.callPackage ../applications/misc/golden-cheetah {};
+  golden-cheetah = golden-cheetah-source;
+
+  golden-cheetah-source = libsForQt5.callPackage ../applications/misc/golden-cheetah {};
+
+  golden-cheetah-app = callPackage ../applications/misc/golden-cheetah-app {};
```

Comment:
Please decide on one packaging method, we don't need both, especially if one is less functional.

Reply:
The reason I decided on two different derivation was the source. The original is build by us from source, the new one is an AppImage where one has to trust the binaries. 
There is also e.g. `firefox-bin` and `firefox` which has a similar case. 

Are we sure we should remove the source package?

Reply:
Both firefox packages are functional but firefox from source performs slightly worse and takes a good while to compile.

Reply:
Done